std/conv.d

```diff
@@ -3967,7 +3967,7 @@ private void emplaceInitializer(T)(ref T chunk) @trusted pure nothrow
     else
     {
         import core.stdc.string : memcpy;
-        static immutable T init = T.init;
+        static immutable T init;// = T.init;
```

Comment:
This fails for types `T` with `@disable this();`, which seems to be the cause of the auto-tester failure